packages/pwa/app/utils/utils.js

```diff
@@ -154,3 +158,85 @@ export const capitalize = (text) => {
         .map((s) => s.charAt(0).toUpperCase() + s.substring(1))
         .join(' ')
 }
+
+export const getAppConfig = () => {
+    return pwaKitConfig.app
+}
+
+/**
+ * get siteId based on the site configuration
+ * @param appConfig
+ * @param originalUrl
+ * @param appOrigin
+ * @returns {string|undefined|*}
+ */
+export const getSiteId = (appConfig = {}, originalUrl, appOrigin) => {
+    const {sites, defaultSiteId} = appConfig
+    let path = originalUrl
+    // If there is no originalUrl value in the locals, create it from the window location.
+    // This happens when executing on the client.
+    if (!path) {
+        path = window?.location.href.replace(window.location.origin, '')
```

Comment:
This line does a hard navigation to home page on client side?

This function is called "getXXX", the navigation is probably unexpected.

Reply:
Another way would be `window.location.pathname`.